### PR TITLE
Lw/camera override

### PR DESCRIPTION
**Pull request by martint-unity**
### Purpose of this PR
Adding the camera override UI / UX functionality.

---
### Release Notes
Updated the UI/UX for the additional camera data for LWRP. 
If user changes a property that can be overridden then we add the component if it doesnt exist.
We also check for post effects layer which supersedes the depth texture.

---
### Testing status
Running Katana now

**Manual Tests**: What did you do?

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low
**Halo Effect**: None, Low, Medium, High?
Low
---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Comment by martint-unity** [comment]
2018-10-25 11_54_08-unity 2019 1 0a6 - preview packages in use - lighting_scene unity - 2019 1 - p

File: com.unity.render-pipelines.lightweight/Editor/LightweightRenderPipelineCameraEditor.cs [hunk 1]
```diff
@@ -116,13 +168,22 @@ void DrawRenderingPath()
             }
         }
 
-        void DrawHDR()
+        public void DrawHDR()
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Editor/LightweightRenderPipelineCameraEditor.cs]: This should not be public

File: com.unity.render-pipelines.lightweight/Editor/LightweightRenderPipelineCameraEditor.cs [hunk 2]
```diff
@@ -116,13 +168,22 @@ void DrawRenderingPath()
...
+            Rect controlRect = EditorGUILayout.GetControlRect(true);
+            EditorGUI.BeginProperty(controlRect, Styles.allowHDR, settings.HDR);
+            int selectedValue = !settings.HDR.boolValue ? 0 : 1;
+            settings.HDR.boolValue = EditorGUI.IntPopup(controlRect, Styles.allowHDR, selectedValue, Styles.displayedCameraOptions, Styles.cameraOptions) == 1;
+            EditorGUI.EndProperty();
+        }
 
-            if (disabled)
-                EditorGUILayout.HelpBox(s_Styles.hdrDisabledWarning, MessageType.Info);
+        public void DrawMSAA()
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Editor/LightweightRenderPipelineCameraEditor.cs]: This as well

File: com.unity.render-pipelines.lightweight/Editor/LWRPAdditionalCameraDataEditor.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+using UnityEditor;
+
+namespace UnityEngine.Experimental.Rendering.LightweightPipeline
```
Comment (phi-lira) [inline com.unity.render-pipelines.lightweight/Editor/LWRPAdditionalCameraDataEditor.cs]: namespace UnityEditor.Experimental... Also remove the using UnityEditor above.

**Comment by martint-unity** [comment]
I have updated with the feedback

**Pull request merged**